cmd/shimdiag/list.go

```diff
@@ -11,15 +13,39 @@ var listCommand = cli.Command{
 	Name:      "list",
 	Usage:     "Lists running shims",
 	ArgsUsage: " ",
-	Before:    appargs.Validate(),
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  "pids",
+			Usage: "Shows the process IDs of each shim",
+		},
+	},
+	Before: appargs.Validate(),
 	Action: func(ctx *cli.Context) error {
+		pids := ctx.Bool("pids")
 		shims, err := findShims("")
 		if err != nil {
 			return err
 		}
+
+		w := new(tabwriter.Writer)
+		w.Init(os.Stdout, 0, 8, 0, '\t', 0)
+
+		if pids {
+			fmt.Fprintln(w, "Shim \t Pid")
+		}
+
 		for _, shim := range shims {
-			fmt.Println(shim)
+			if pids {
+				pid, err := getPid(shim)
+				if err != nil {
+					return err
+				}
+				fmt.Fprintf(w, "%s \t %d\n", shim, pid)
+			} else {
+				fmt.Println(shim)
```

Comment:
Do you need to use the tabwriter here?

Reply:
Nope, here's the output of both:

PS C:\ContainerPlat> .\shimdiag.exe list
k8s.io-10ca615f457902b0a31252ad299f64bd22e0a09b309eb36e0bf5b61376f563b1
k8s.io-128d6078c12b8df8a219ef3569b7542f4d19cc3615f7ca7f130724a911e92d21
k8s.io-f5b13b1e43e3223f3591a5a4e20938643453a9fd88ad21eec0274e4b600abcc5

PS C:\ContainerPlat> .\shimdiag.exe list -pids
Shim                                                                                                                                Pid
k8s.io-10ca615f457902b0a31252ad299f64bd22e0a09b309eb36e0bf5b61376f563b1  48420
k8s.io-128d6078c12b8df8a219ef3569b7542f4d19cc3615f7ca7f130724a911e92d21    30272
k8s.io-f5b13b1e43e3223f3591a5a4e20938643453a9fd88ad21eec0274e4b600abcc5  17992

Reply:
Lovely formatting github, here's a pic instead. 
![listpids](https://user-images.githubusercontent.com/36526702/105093992-70249680-5a58-11eb-857f-95536b894bdb.PNG)

Reply:
What's the point of making the tab writer then when pids is not specified? Might as well use it if it doesn't cause fformat issues

Reply:
done

Reply:
No reason other than we have to make it before because we have to flush it after the loop so it can't be made inside.